qunicorn_core/api/api_models/job_dtos.py

```diff
@@ -113,19 +113,18 @@ class JobExecutePythonFileDto:
 
 
 class JobRequestDtoSchema(MaBaseSchema):
-    name = ma.fields.String(required=True, example="JobName")
-    provider_name = ma.fields.Enum(required=True, example=ProviderName.IBM, enum=ProviderName)
-    device_name = ma.fields.String(required=True, example="aer_simulator")
+    name = ma.fields.String(required=True, metadata={"example": "JobName"})
+    provider_name = ma.fields.Enum(required=True, metadata={"example": ProviderName.IBM}, enum=ProviderName)
+    device_name = ma.fields.String(required=True, metadata={"example": "aer_simulator"})
     shots = ma.fields.Int(
         required=False,
         allow_none=True,
-        metadata={"label": "Shots", "description": "Number of shots", "input_type": "number"},
-        example=4000,
+        metadata={"example": 4000, "label": "Shots", "description": "Number of shots", "input_type": "number"},
```

Comment:
das ist jetzt nicht von der Änderung in dem PR, aber hier könnte man sich auf eine Schreibweise der Parameter festlegen (groß/klein schreibung)
```suggestion
        metadata={"example": 4000, "label": "shots", "description": "number of shots", "input_type": "number"},
```